server/api/attendance/index.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var express = require('express');
+var controller = require('./attendance.controller');
+var config = require('../../config/environment');
+var auth = require('../../auth/auth.service');
+
+var router = express.Router();
+
+// Basic Access / Manipulation
+router.get('/', auth.hasRole('mentor'), controller.index);
+router.get('/:id', auth.isAuthenticated(), controller.show);
+router.delete('/:id', auth.hasRole('mentor'), controller.destroy);
+
+// DON'T USE : does not generate required data for submission
+router.post('/', auth.hasRole('admin'), controller.create);
+router.put('/:id', auth.hasRole('admin'), controller.update);
+
+// Verifies existing attendance submissions in the DB.
+router.put('/:id/verify', auth.hasRole('mentor'), controller.verifyAttendanceById);
+router.post('/present/:user/full', auth.hasRole('mentor'), controller.verifyFullAttendance);
+router.post('/present/:user/small', auth.hasRole('mentor'), controller.verifySmallAttendance);
+
+// Get attendance for a specific user (or current user) [on a date]
+router.get('/present/me/:date', auth.isAuthenticated(), controller.presentMe);
+router.get('/present/:user/:date', auth.hasRole('mentor'), controller.present);
+router.get('/present/me', auth.isAuthenticated(), controller.getAttendanceMe);
+router.get('/present/:user', auth.hasRole('mentor'), controller.getAttendance);
+
+// Mark attendance as present, subject to verification
+router.post('/attend', auth.isAuthenticated(), controller.attend);
+
+// Set attendance as present (no verification)
+router.post('/attend/small/:user', auth.hasRole('mentor'), controller.setAttendanceSmall);
```

Comment:
why do mentors have 4 different api endpoints but the user side of attend is only one? can't we slim this down?